src/syscheckd/registry/registry.c

```diff
@@ -14,7 +14,6 @@
 #include "shared.h"
 #include "../syscheck.h"
 #include "../db/fim_db.h"
-#include "../db/fim_db_registries.h"
```

Comment:
Excluding this header file causes many `implicit definition` warnings during compilation, please readd it.